File: baekjoon/bronze/17224/17224.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	var n, l, k, ans int
	var easy, hard int
	var scores []int

	fmt.Scan(&n, &l, &k)
	for i := 0; i < n; i++ {
		fmt.Scan(&easy, &hard)
		if l < easy {
			scores = append(scores, 0)
		} else if l >= hard {
			scores = append(scores, 140)
		} else {
			scores = append(scores, 100)
		}
	}
	sort.Slice(scores, func(i, j int) bool {
		return scores[i] > scores[j]
	})
	for i := 0; i < k; i++ {
		ans += scores[i]
	}
	fmt.Println(ans)
}

File: baekjoon/bronze/17224/17224.cpp
#include <iostream>

int min(int a, int b);

int main() {
  int n, l, k, ans;
  int easy, easy_cnt, hard, hard_cnt;
  
  easy_cnt = 0; hard_cnt = 0;
  scanf("%d %d %d\n", &n, &l, &k);
  for (int i = 0; i < n; i++) {
    scanf("%d %d\n", &easy, &hard);
    if (hard <= l) {
      hard_cnt++;
    } else if (easy <= l) {
      easy_cnt++;
    }
  }
  ans = min(hard_cnt, k) * 140;
  k -= min(hard_cnt, k);
  ans += min(easy_cnt, k) * 100;
  printf("%d\n", ans);

  return 0;
}

int min(int a, int b) {
  return a < b ? a : b;
}